Polyhedron/demo/Polyhedron/Scene_c3t3_item.h

```diff
@@ -59,6 +59,8 @@ using namespace CGAL::Three;
     }
     bool load_binary(std::istream& is) override;
 
+    void compute_bbox() const Q_DECL_OVERRIDE;
```

Comment:
Since C++11 we use `override` directly, especially if other methods of the file already use it:
```suggestion
    void compute_bbox() const override;
```